platform/broadcom/sonic-platform-modules-dell/z9100/scripts/z9100_platform.sh

```diff
@@ -171,6 +192,13 @@ if [[ "$1" == "init" ]]; then
     echo $value > /sys/class/i2c-adapter/i2c-15/15-003e/qsfp_lpmode
     echo $value > /sys/class/i2c-adapter/i2c-16/16-003e/qsfp_lpmode
 
+
+    #Copy led_proc_init.soc
+    switch_port_led
+    if [ -e $led_proc_init ] && [ ! -e $device/$platform/led_proc_init.soc ]; then
+      cp $led_proc_init $device/$platform
```

Comment:
I'm not a fan of copying the file from the HwSKU directory into to the platform directory. If the LED functionality need to differ based on SKU, then we may want to consider moving all led_proc_init.soc files into the respective HwSKU directories. For SKUs which share a common led_proc_init.soc file, we can symlink them.

I'm not sure that this should hold up this PR, but it's something I wanted to at least note here for posterity.

Reply:
Joe, thanks for the view. I changed copy to symlink in new commit. Right now as per the design ledinit "platform/broadcom/docker-syncd-brcm/supervisord.conf" picks led_proc_init.soc in platform specific locaation than HWSKU

**command=/usr/bin/bcmcmd -t 60 "rcload /usr/share/sonic/platform/led_proc_init.soc"**

Reply:
Yes, I understand that the current implementation looks for the led_proc_init.soc file in the platform directory. What I was noting for the future is that maybe we should consider moving the files into HwSKU directories and symlinking them if they share the same file.

I wasn't asking you to replace your copy operation with symlinking, but I'm OK with the change. My comment was about refactoring the mechanism completely in the future.

Reply:
Thanks Joe. Got it!